spec/feature/mission_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe 'Missions List', type: :feature do
+  subject { page }
+  
+  let(:mission) { create(:mission) }
+
+  describe 'Showing missions' do
+    context 'when there is no misson' do
+      before { visit missions_path }
+      
+      it_behaves_like 'a missions list page'
+    end
+
+    context 'when there are missions' do
+      before do
+        mission
+        visit missions_path
+      end
```

Comment:
為了明確表達你的意圖，請改用類似以下的寫法：
```suggestion
      let!(:mission) { create(:mission) }

      before { visit missions_path }  
```